fe/fe-core/src/main/java/com/starrocks/lake/StarMgrMetaSyncer.java

```diff
@@ -126,7 +126,7 @@ public static void dropTabletAndDeleteShard(List<Long> shardIds, StarOSAgent sta
                 LakeService lakeService = BrpcProxy.getLakeService(node.getHost(), node.getBrpcPort());
                 DeleteTabletResponse response = lakeService.deleteTablet(request).get();
                 if (response != null && response.failedTablets != null && !response.failedTablets.isEmpty()) {
-                    TStatusCode stCode = TStatusCode.findByValue(response.getStatus().getStatusCode());
+                    TStatusCode stCode = TStatusCode.findByValue(response.status.statusCode);
```

Comment:
what difference between the code after changing and the previous?

Reply:
no difference, looks like in 3.2.x/3.1.x branch, the code is generated without `getStatus()`/`getStatusCode()` interface, need to access the member variable directly. This change is just to keep the same as in backport branch.

The UT coverage is a more valuable change in this PR.